### ISSUE TITLE
Upgrading the Cassandra Java Driver to 3.10.x

### ISSUE THREAD

**Issue by Sfurti-yb** [yugabyte/yugabyte-db#9785]
For the Cassandra Java Driver upkeep and upgrade from 3.8.x to 3.10.x:

1. The steps followed and the changes made :  
    [Google doc](https://docs.google.com/document/d/1FZKriCrf_ypJAef83a8_nWjK1vbcf0z6nIEFQvuuNU4/edit?usp=sharing) 
2. Link to the commit:
    [Link](https://github.com/yugabyte/cassandra-java-driver/commit/72610157126d3adfd2ca35cfe19c747ce74d8429)

**Comment by nchandrappa** [comment]
Commit LGTM. 

We will need to rename the snapshot versions to `3.10.3-yb-1-SNAPSHOT` instead of `3.10.3-yb-8-SNAPSHOT`.  `yb-number` represents the build number, since this will be the first build on maven central we need to have `3.10.3-yb-1-SNAPSHOT`. Please update the version before releasing the jar.

Release the jar with `3.10.3-yb-1-RC1` tag.

**Comment by Sfurti-yb** [comment]
Released the jar with 3.10.3-yb-1-RC1 tag.
``` 
<dependency>
  <groupId>com.yugabyte</groupId>
  <artifactId>cassandra-driver-core</artifactId>
  <version>3.10.3-yb-1-RC1</version>
</dependency> 
```
